pantheon.php

```diff
@@ -3,14 +3,14 @@
  * Plugin Name: Pantheon
  * Plugin URI: https://pantheon.io/
  * Description: Building on Pantheon's and WordPress's strengths, together.
- * Version: 1.2.0
+ * Version: 1.2.1
  * Author: Pantheon
  * Author URI: https://pantheon.io/
  *
  * @package pantheon
  */
 
-define( 'PANTHEON_MU_PLUGIN_VERSION', '1.2.0' );
+define( 'PANTHEON_MU_PLUGIN_VERSION', '1.2.1' );
```

Comment:
So no code actually changed. Do we need to cut a release?

Reply:
I suppose since upstream goes off master we're basically cutting a release anyway. I'd love to see upstream pull come off the latest release instead of `main` in the future.

Reply:
we could not. the jira ticket specifically says we should cut a release but we can opt against since we didn't change the mu plugin itself. there's no standard/codeowners process on versioning releases, either, so we could bump the version in the files without actually making any changes, with the assumption that the next time we do change something, it will be part of 1.2.1.